src/components/GenericAccount.tsx

```diff
@@ -0,0 +1,107 @@
+// Copyright 2021 Parity Technologies (UK) Ltd.
+// This file is part of Parity Bridges UI.
+//
+// Parity Bridges UI is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Parity Bridges UI is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Parity Bridges UI.  If not, see <http://www.gnu.org/licenses/>.
+
+import Container from '@material-ui/core/Container';
+import { encodeAddress } from '@polkadot/util-crypto';
+import React from 'react';
+import { useUpdateTransactionContext } from '../contexts/TransactionContext';
+import { TransactionActionCreators } from '../actions/transactionActions';
+import { getChainConfigs } from '../configs/substrateProviders';
+import { makeStyles } from '@material-ui/core/styles';
+import { useSourceTarget } from '../contexts/SourceTargetContextProvider';
+import useBalance from '../hooks/useBalance';
+import useReceiver from '../hooks/useReceiver';
+import getDeriveAccount from '../util/getDeriveAccount';
+import AccountDisplay from './AccountDisplay';
+
+interface Props {
+  value: string;
+  isDerived?: boolean;
+}
+
+const useStyles = makeStyles(() => ({
+  container: {
+    padding: '0',
+    float: 'left'
+  },
+  native: {
+    border: '1px solid',
+    borderTop: 'none',
+    padding: '5px 0'
+  },
+  companion: {
+    border: '1px solid',
+    borderTop: 'none',
+    borderRadius: '0 0 5px 5px',
+    padding: '5px 0'
+  }
+}));
+
+const GenericAccount = ({ value }: Props) => {
```

Comment:
Please remove if this is unused.

Reply:
i prefer this component to stay even if it is not used. Removing it will lead us to work on something that needs adjustments not recreating the component from scratch.

Reply:
No, please remove. It will make it harder to review in the future and dead code should not be in the repository.